tests/framework/db/mysql/CommandTest.php

```
@@ -8,19 +8,80 @@
+#[Group('mysql')]
+#[Group('command')]
+final class CommandTest extends BaseCommand
 {
     public $driverName = 'mysql';
 
     protected $upsertTestCharCast = 'CONVERT([[address]], CHAR)';
 
+    #[DataProviderExternal(CommandProvider::class, 'addCommentOnColumn')]
+    public function testAddUpdateDropCommentOnColumn(string $tableName, string $commentTarget, string $columnName): void
+    {
+        $db = $this->getConnection(false);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)